packages/vaadin-login/test/login-form.test.js

```diff
@@ -187,21 +186,21 @@ describe('login form', () => {
     enter(vaadinLoginPassword);
     expect(submitStub.called).to.be.false;
 
-    tap(submit);
+    click(submit);
```

Comment:
question: not clear why a `tap` is replaced by a `click`. was it intentional?

Reply:
Restored.